### ISSUE TITLE
Accessibility basics missing

### ISSUE THREAD

**Issue by nilmerg** [Icinga/icingadb-web#186]
Even the most basic accessibility expectations such as titles for links are missing. We should add them...

**Comment by Yonas-net** [comment]
This [html element](https://github.com/Icinga/icingaweb2/blob/master/library/Icinga/Web/Widget/Tabs.php#L313-L316) is not visible in Icinga Web 2. I just noticed it when the screen reader read out the titles. So I really don't know whether we really need this if you can't see the cancel icon at all.

**Comment by nilmerg** [comment]
It's an interactive element, that should *probably* be visible for screen readers. If it's not, put it on the list of things you think need to be changed.

**Comment by Yonas-net** [comment]
I know it has nothing to do with this issue at all but I didn't see it before. Isn't that kind of weird.

<img width="795" alt="Bildschirmfoto 2021-08-18 um 15 40 03" src="https://user-images.githubusercontent.com/57616252/129908745-fa067b75-38e2-49fd-bf54-9d7614cb0fe3.png">
